include/FreeRTOS_IP_Private.h

```diff
@@ -900,22 +833,9 @@
  * if they are not being used anywhere. But their use depends on the
  * application and hence these functions are defined unconditionally.
  */
-    static portINLINE uint32_t ulChar2u32( const uint8_t * apChr );
-    static portINLINE uint32_t ulChar2u32( const uint8_t * apChr )
-    {
```

Comment:
/md The parameter name `apChr` is in C++ style, it should be `pucPtr`, See functions `usChar2u16()` and `ulChar2u32()`, also in `FreeRTOS_IP_Private.h` please.